src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -72,9 +75,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     // - <none>
     void TermControlAutomationPeer::SignalCursorChanged()
     {
+        UiaTracing::Signal::CursorChanged();
         Dispatcher().RunAsync(Windows::UI::Core::CoreDispatcherPriority::Normal, [&]() {
             // The event that is raised when the text was changed in an edit control.
-            RaiseAutomationEvent(AutomationEvents::TextEditTextChanged);
+            RaiseAutomationEvent(AutomationEvents::TextPatternOnTextSelectionChanged);

```

Comment:
Just curious, what does this event mean, and why was `TextEditTextChanged` the wrong one?

Reply:
TextEditChanged is apparently for composition. Though you _can_ send a TextEditTextChanged event on it's on, the first question is "was it compositionCompleted, started, or what?". For whatever known reason, if that info isn't provided, some other apps still try to read it and get a nullptr exception.

NVDA does not make this mistake so props to them haha.

Reply:
I can't +1 this enough